src/opts.rs

```diff
@@ -248,6 +252,17 @@ pub enum RedundantLabels {
     Strip,
 }
 
+#[derive(Debug, Copy, Clone, Bpaf, Eq, PartialEq)]
+#[bpaf(fallback(Demangle::Yes))]
+pub enum Demangle {
```

Comment:
How does demangle interacts with `full_name`? Does it make sense to combine it with `Demangle` and have something like this:

```rust
#[bpaf(fallback(NameDisplay::Short))]
enum NameDisplay {
    /// Include full demangled name instead of just prefix
    FullName,

    ShortName,

    /// Do not demangle symbol names
    #[bpaf(long("keep-mangled"))]
    Mangled
}
```

Reply:
Right now demangle overrides `full_name`, so e.g. `--keep-mangled --full-name` is equivalent to `--keep-mangled`.  I think it makes sense to combine them though, that's a good idea. :)

Reply:
Good, it's no longer crashing :) Can you please rename `--short` to `--short-name` with `#[bpaf(long("keep-mangled"))]`? Did you check that no-mangle code works for you? Does it makes sense to add a CI case for that?

Reply:
(or make --short` hidden with `#[bpaf(skip)]` - I'm not sure how useful that is if it's a default behavior)

Reply:
Yup, it works just fine for me. I'll add a test case on CI and rename `short` param.